### ISSUE TITLE
Regarding selection of color space in the implementation of BlazePose model

### ISSUE THREAD

**Issue by Debosmit-Neogi** [google/mediapipe#2149]
I got better results in terms of visibility scores using Lab color space instead of using RGB for most of the landmarks. The visibility scores for 5 random landmark points  using Lab and RGB is enumerated below

Landmark      RGB           Lab
0                    0.994         0.9996
1                    0.996         0.9997
2                    0.995         0.9998
25                  0.815         0.6874
32                  0.348         0.3558

**Comment by sgowroji** [comment]
Hi @Debosmit-Neogi, Could you please let us know what exactly your query with respect to the above context. Thanks!

**Comment by Debosmit-Neogi** [comment]
Sir @sgowroji I have used the mediapipe package in python for pose estimation.  In one step we have to convert the colour space from BGR to RGB as openCV processes the visual inputs in BGR colour space as it is recommended in the official documentation. But I've used Lab colour space instead of RGB and got better landmark visibility scores. My query is if this observation is true for certain cases or is it generalised?
